static/scss/detail/learning-techniques.scss

```diff
@@ -0,0 +1,75 @@
+// sass-lint:disable mixins-before-declarations
+@import "common";
+
+.learning-techniques-block {
+  padding-top: 70px;
+  padding-bottom: 50px;
+
+  h1 {
+    text-align: center;
+    font-size: 40px;
+    line-height: 44px;
+    margin: 0 0 50px;
+    color: $primary;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+}
+
+.learning-techniques-list {
+  margin: 0;
+  overflow: hidden;
```

Comment:
Can you include a short comment above this style definition? Something like `// This is here to hide the outer list item borders`